src/Afsy/Bundle/CoreBundle/Entity/Article.php

```diff
@@ -74,6 +74,8 @@ class Article implements Taggable
     /**
      * @var \DateTime
      * @ORM\Column(type="datetime", name="event_date", nullable=true)
+     * @Assert\DateTime()
+     * @Assert\GreaterThan("today")
```

Comment:
I'm not in favor of this, insofar of I remember a few times we forgot to create an article for an event so we do it after it actually happened.
This assertion is a restriction that brings nothing important and that could be potentially annoying IMHO.

Reply:
It's made to avoid errors. Could be annoying indeed.